yarn.lock

```diff
@@ -177,6 +177,7 @@ __metadata:
   version: 0.0.0-use.local
   resolution: "root@workspace:."
   dependencies:
+    "@sentry/node": 7.14.1
```

Comment:
@almeidx this is the problematic line, right?

Reply:
Yep